### PR TITLE
Rename `myModal` to `eventDetailBackground` in meetings schedule

### PR DESCRIPTION
**Pull request by Czaki**
# References and relevant issues

https://github.com/napari/docs/pull/337#discussion_r1472646332

# Description

Rename to increase code readability

### CODE REVIEW

**Review by jni — APPROVED** [review]

File: docs/community/meeting_schedule.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ If you are using napari or interested in how napari could be used in your work,
 
 <div id='timezone'></div>
 
-<div id="myModal" class="modal">
+<div id=" eventDetailBackground" class="modal">
```
Comment (jni) [inline docs/community/meeting_schedule.md]:
```suggestion
<div id="eventDetailBackground" class="modal">
```

File: docs/community/meeting_schedule.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ If you are using napari or interested in how napari could be used in your work,
         // Get the <span> element that closes the modal
         var span = document.getElementsByClassName("close")[0];
         // Get the modal
-        var modal = document.getElementById("myModal");
+        var modal = document.getElementById(" eventDetailBackground");
```
Comment (jni) [inline docs/community/meeting_schedule.md]:
```suggestion
        var modal = document.getElementById("eventDetailBackground");
```

File: docs/community/meeting_schedule.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ If you are using napari or interested in how napari could be used in your work,
         }
         // When the user clicks anywhere outside of the modal, close it
         modal.onclick = function(event) {
-          if (event.target.id == "myModal") {
+          if (event.target.id == " eventDetailBackground") {
```
Comment (jni) [inline docs/community/meeting_schedule.md]:
```suggestion
          if (event.target.id == "eventDetailBackground") {
```

**Review by jni — APPROVED** [review]

**Comment by jni** [comment]
@Czaki I think the space at the start of the id is a typo? I've made suggestions to remove it. But, merge when ready. 😊

**Review by melissawm — APPROVED** [review]
Thank you, I might need to organize this code better in the future but this is already helpful. Cheers!

**Pull request merged**